ChatSchema.md

```diff
@@ -0,0 +1,105 @@
+# Chat Server-side Schema
+
+## Entity Design
+
+### User
+
+User entity includes:
+
+- user_id: identify a user. (Pk)
+- name: shown name.
```

Comment:
Is it possible to use Google account email address as the primary key? So that we can re-use the authentication provided by Google API, and save a registry step for users.